### PR TITLE
Add node dependencies to discovery role

### PR DESCRIPTION
**Pull request by bderusha**
@clintonb please review

### CODE REVIEW

File: playbooks/roles/discovery/tasks/main.yml [hunk 1]
```diff
@@ -76,6 +76,35 @@
     - devstack
     - devstack:install
 
+- name: Create nodeenv
```
Comment (clintonb) [inline playbooks/roles/discovery/tasks/main.yml]: There is another issue afoot. `nodeenv` needs to be installed by the call to `make production-requirements`; however, the call to `make requirements` will attempt to use npm. You will need to modify the Makefile in course-discovery.

See #3261 and https://github.com/edx/course-discovery/pull/172.

**Comment by bderusha** [comment]
FYI @feanil

**Comment by bderusha** [comment]
@clintonb

**Comment by clintonb** [comment]
👍

**Pull request merged**